### PR TITLE
[cleanup] [trivial] Change some functions to return bool instead of int

### PR DESCRIPTION
**Pull request by yebblies**

### CODE REVIEW

File: src/access.c [hunk 1]
```diff
@@ -194,6 +191,8 @@ void accessCheck(AggregateDeclaration *ad, Loc loc, Scope *sc, Dsymbol *smember)
     // BUG: should enable this check
     //assert(smember->parent->isBaseOf(this, NULL));
 
+    bool result;
+    Prot access;
```
Comment (AndrejMitrovic) [inline src/access.c]: Unused?

Comment (yebblies) [inline src/access.c]: Unused above this line, it's set in the if condition below.

Comment (AndrejMitrovic) [inline src/access.c]: Oh you moved it from above, I missed it. :)

**Comment by AndrejMitrovic** [comment]
Auto-merge toggled on

**Pull request merged**